feature-libs/product-multi-dimensional/selector/components/product-multi-dimensional-selector-components.module.ts

```diff
@@ -5,9 +5,22 @@
  */
 
 import { NgModule } from '@angular/core';
+import { CmsConfig, provideDefaultConfig } from '@spartacus/core';
 import { ProductMultiDimensionalSelectorModule } from './selector/product-multi-dimensional-selector.module';
+import { ProductMultiDimensionalSelectorComponent } from './selector/product-multi-dimensional-selector.component';
+import { ProductMultiDimensionalSelectorGuard } from './guards';
 
 @NgModule({
   imports: [ProductMultiDimensionalSelectorModule],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
```

Comment:
`major:` I'd rather move this config to the `ProductMultiDimensionalSelectorModule` -`ProductMultiDimensionalSelectorComponent` and `ProductMultiDimensionalSelectorGuard` are declared and provided there and such module can also encapsulate them together with the config. The components module doesn't have to have a notion about this.

Reply:
Sure, I have seen features implemented both ways. Additionally, I've changed the module name as it was confused for a moment:
`ProductMultiDimensionalSelectorModule` ->`ProductMultiDimensionalSelectorComponentModule`